account-decoder/src/parse_stake.rs

```diff
@@ -19,7 +19,7 @@ pub fn parse_stake(data: &[u8]) -> Result<StakeAccountType, ParseAccountError> {
             meta: meta.into(),
             stake: None,
         }),
-        StakeState::Stake(meta, stake) => StakeAccountType::Delegated(UiStakeAccount {
+        StakeState::Stake(meta, stake, _) => StakeAccountType::Delegated(UiStakeAccount {
```

Comment:
Nit-can-be-done-after-this-PR: can you add the flag to `UiStakeAccount` so that the flag is printed if it's not `Empty`?

Reply:
yeah. will do it in a follow up pr.

Reply:
https://github.com/solana-labs/solana/pull/32033